.github/ISSUE_TEMPLATE/REPORT_A_BUG.yml

```
@@ -0,0 +1,76 @@
+name: '🐛 Report a bug'
+description: 'Is something not working as you expect?'
+labels: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to file a bug report!
```

```suggestion
        Thanks for taking the time to file a bug report!
        
        Before posting, please check that the bug hasn't already been:
        -  fixed in the next release (https://github.com/stylelint/stylelint/blob/main/CHANGELOG.md)
        -  discussed previously (https://github.com/stylelint/stylelint/search)

       You can help us fix the bug more quickly by:
       1. Figuring out what needs doing and proposing it.
       2. Writing the code and submitting a PR once the bug is confirmed.
```

Let's add this blurb back in as it:
- reduces duplicates
- encourages people to get involved

---

Done! e516da71